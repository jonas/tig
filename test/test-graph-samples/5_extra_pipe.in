commit A B C
    Commit A - merge B and C
commit C D
    Commit C after D
commit B E D
    Commit B - merge E and D
commit D F
    Commit D after F
commit E
    Commit E
commit F
    Commit F
