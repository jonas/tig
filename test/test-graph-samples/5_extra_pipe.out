●─╮ Commit A - merge B and C
│ ● Commit C after D
●─│─╮ Commit B - merge E and D
│ ●─╯ Commit D after F
◎ │ Commit E
◎─╯ Commit F
